src/commands/txCommands/util/index.js

```diff
@@ -181,7 +181,9 @@ const readResource$ = (slug, project = PROJECT) =>
 	Rx.Observable.bindNodeCallback(tx.sourceLanguageMethods.bind(tx))(
 		project,
 		slug
-	).retry(5);
+	)
+	.do(null, () => console.log(`error readResource$ ${slug} ${project}`))
```

Comment:
i think the error handling function here would actually get an `err` argument - would it be worth logging that as well or are we guaranteed to log the error elsewhere in the chain?

Reply:
We could log it but i'm not sure if it would be useful. i've decided not to worry about all the things that might be useful and just log the things that were useful but i can field requests.

In the work i did, it was important to know the call was failing but available error info wasn't useful and a bit misleading.